src/uct/api/uct.h

```diff
@@ -2693,6 +2873,70 @@ UCT_INLINE_API unsigned uct_iface_progress(uct_iface_h iface)
 }
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Open a connection manager.
+ *
+ * Open a specific connection manager. All client server connection
+ * establishment operations are performed in the context of a specific
+ * connection manager.
+ * @note This is an alternative API for
+ *       @ref uct_iface_open_mode::UCT_IFACE_OPEN_MODE_SOCKADDR_SERVER and
```

Comment:
should be mark that API as deprecated, and replaced by uct_cm_open?

Reply:
IMO not yet. Maybe after we have an implementation for the new API?

Reply:
ok